README.md

```diff
@@ -1,115 +1,176 @@
-# NyaaV2 [![Build Status](https://travis-ci.org/nyaadevs/nyaa.svg?branch=master)](https://travis-ci.org/nyaadevs/nyaa)
+# Mogu!
+
+This is a fork of [Nyaa](https://github.com/nyaadevs/nyaa) for HoloPirates.
 
 ## Setting up for development
-This project uses Python 3.7. There are features used that do not exist in 3.6, so make sure to use Python 3.7.
-This guide also assumes you 1) are using Linux and 2) are somewhat capable with the commandline.   
-It's not impossible to run Nyaa on Windows, but this guide doesn't focus on that.
+
+This project uses Python 3.7. There are features used that do not exist in 3.6,
+so make sure to use Python 3.7. This guide also assumes you 1) are using Linux
+and 2) are somewhat capable with the command line. It's not impossible to run
+Nyaa on Windows, but this guide doesn't focus on that.
 
 ### Code Quality:
-- Before we get any deeper, remember to follow PEP8 style guidelines and run `./dev.py lint` before committing to see a list of warnings/problems.
-    - You may also use `./dev.py fix && ./dev.py isort` to automatically fix some of the issues reported by the previous command.
-- Other than PEP8, try to keep your code clean and easy to understand, as well. It's only polite!
+
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+
+We use [Black](https://github.com/psf/black) to automatically format all code.
```

Comment:
I'm really confused, here it says you use black, but in this PR you also add a .pretterrc.json and format everything with said prettierrc. Could it be made clearer which one is actually used for this project?